.github/workflows/bottest.yml

```diff
@@ -0,0 +1,127 @@
+on:
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: 'Pull Request Number'
+        required: false
+        default: ''
+
+  issue_comment:
+    types: [created]
+
+name: Benchmarks
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_REPO: ${{ github.repository }}
+
+jobs:
+  benchmark:
+    name: run 30-bot bot test
```

Comment:
nit: for some reason my brain struggled to parse this - any opinion on "run test (30 bots)" or something?